tests/Jackalope/Tools/Console/InitDoctrineDbalCommandTest.php

```diff
@@ -45,7 +45,7 @@ public function setUp(): void
         $this->connection = $this->createMock(Connection::class);
         $this->schemaManager = $this->createMock(AbstractSchemaManager::class);
 
-        $this->platform = $this->createMock(MySqlPlatform::class);
+        $this->platform = $this->createMock(MySQLPlatform::class);
```

Comment:
the case change seems to happen later, there is a warning in the lowest version build: https://travis-ci.com/github/jackalope/jackalope-doctrine-dbal/jobs/454621052

dbal 2 seems to still have it lowercase. would it make sense to define an alias if the uppercase version does not exist?

Reply:
I was really short of good ideas here. Unfortunately `MySqlPlatform` and `MySQLPlatform` are exactly the same token for PHP, as class names are case-insensitive. The problem here is `require` the correct file on case-sensitive filesystems.

I've added a `dbal2_compat.php` file which adds a simple autoloader function in case the uppercase-named class was not present, but I'm unsure about that. What do you think about it?

Reply:
clever solution! i asked on twitter if there is any best practice but at least this would work.

Reply:
on twitter, somebody said it did [not work for him](https://twitter.com/rpkamp82/status/1335573797383139329), guessing that class_exists within an autoloader does not attempt an autoload to prevent endless loops.

makes me wonder if it would be enough to have something (outside the autoloaders) that does `class_exists(MySQLPlatform::class) || class_exists(MySqlPlatform::class)`? that dbal2_compat.php could maybe have just those 2 lines, no? if it is always executed, it will load the class if it exists at all and then case sensitivity of the filesystem no longer matters.

Reply:
Mh, that's weird, probably WSL is involved in this error. Anyway I updated the code to call two class_exists in the compat file as you suggested.